lib/util/Components.js

```diff
@@ -114,8 +114,12 @@ Components.prototype.list = function() {
     }
     const id = this._getId(this._list[j].node);
     list[j] = this._list[j];
+    if (!list[j].usedPropTypes) {
+      list[j].usedPropTypes = [];
+    }
+
     if (usedPropTypes[id]) {
-      list[j].usedPropTypes = (list[j].usedPropTypes || []).concat(usedPropTypes[id]);
```

Comment:
I see what you mean. Could you point to the other lines you'd need an `|| []` on, if you reverted these lines?